test/NuGet.Core.Tests/NuGet.Commands.Test/SignCommandRunnerTests.cs

```diff
@@ -109,7 +109,8 @@ public async Task ExecuteCommandAsync_WithAmbiguousMatch_ThrowsAsync()
             using (var test = await Test.CreateAsync(_fixture.GetDefaultCertificate()))
             {
                 test.Args.CertificateSubjectName = "Root";
-                test.Args.CertificateStoreLocation = CertificateStoreUtilities.GetTrustedCertificateStoreLocation();                
+                //X509 store is opened in ReadOnly mode in this code path. Hence StoreLocation is set to LocalMachine.
+                test.Args.CertificateStoreLocation = CertificateStoreUtilities.GetTrustedCertificateStoreLocation(readOnly: true);         
```

Comment:
So this test is trying to search the specified certificate store by subject name. And it assumed to have multiple certificates in the search results. But CurrentUser/Root doesn't have any certificate unless we add certificate first. While LocalMachine/Root has lots of certificate by default, so that's why we use LocalMachine/Root in this test case. Is that correct?

Reply:
@heng-liu - That is correct.